downstream/modules/platform/ref-ldap-config-on-pah.adoc

```diff
@@ -0,0 +1,177 @@
+:_content-type: REFERENCE
+
+[id="ref-ldap-config-on-pah_{context}"]
+= LDAP configuration on {PrivateHubName}
+
+You must set the following six variables in your {PlatformName} installer inventory file to configure your {PrivateHubName} for LDAP authentication:
```

Comment:
@sayjadha - Two things with this one since I'm decided to leave it as is: 

1. SMEs (especially the one from Support who reported that we needed to create these instructions) specified that we mention these specific variables. All the other variables can be left as is or updated as required by the customer.
2. Since this has already been SME and peer reviewed, we will leave this as is. We can come back to it at a later date if it requires updates.